rust-toolchain.toml

```
@@ -1,5 +1,5 @@
 [toolchain]
-channel = "1.80"
+channel = "1.81"
 profile = "minimal"
 components = [ "rustfmt", "clippy" ]
 targets = [ "x86_64-apple-darwin", "aarch64-apple-darwin", "x86_64-unknown-linux-gnu", "wasm32-wasi", "x86_64-pc-windows-msvc" ]
```

```suggestion
targets = [ "x86_64-apple-darwin", "aarch64-apple-darwin", "x86_64-unknown-linux-gnu", "wasm32-wasip1", "x86_64-pc-windows-msvc" ]
```

I believe this target should be changed too, as the [WASI 0.1 target naming changed](https://blog.rust-lang.org/2024/09/05/Rust-1.81.0.html#wasi-01-target-naming-changed).

This might however require some [changes in other files aswell](https://github.com/search?q=repo%3Azed-industries%2Fzed+wasm32-wasi&type=code).

---

Thanks!